auraed/Cargo.toml

```diff
@@ -65,6 +69,7 @@ procfs = "0.14.2"
 rtnetlink = "0.11.0"
 serde_json.workspace = true
 serde.workspace = true
+signal = "0.5.1"
```

Comment:
Is this dependency being used? There is a newer version of it also, is there a reason to use the older version? If we only want a signal enum, nix has one at `nix::sys::signal::Signal`